src/Microsoft.ML.Data/Scorers/GenericScorer.cs

```diff
@@ -225,7 +225,12 @@ void ISaveAsPfa.SaveAsPfa(BoundPfaContext ctx)
         void ISaveAsOnnx.SaveAsOnnx(OnnxContext ctx)
         {
             Host.CheckValue(ctx, nameof(ctx));
+
+            const int minimumOpSetVersion = 9;
+            ctx.CheckOpSetVersion(minimumOpSetVersion, LoaderSignature);
+
             Host.Assert(Bindable is IBindableCanSaveOnnx);
```

Comment:
Is this necessary here? 
As a rule of thumb, if a function does not contain any references to onnx operators it is not a good candidate for determining the opset version to use. If we are going against that rule of thumb, it would be good to know why we are doing it.

Reply:
This is something I missed(I initially assume almost every transformer uses op). Yep I'll follow the rule that only check op version when the transformer uses it